frontend/charm/requirements.txt

```diff
@@ -1 +1,2 @@
-ops >= 2.2.0
+ops >= 2.8.0
+macaroonbakery==1.3.2
```

Comment:
Added as a workaround to https://github.com/canonical/test_observer/actions/runs/7142353669/job/19451476954 as instructed at https://chat.canonical.com/canonical/pl/9xzxm61rgtry5d7sfx9xz7ryyc